script/e2e-test.sh

```diff
@@ -67,12 +67,25 @@ k8s_ensure_image kubeapps kubeapps-ci-internal-dashboard $DEV_TAG
 k8s_ensure_image kubeapps kubeapps-ci-internal-tiller-proxy $DEV_TAG
 
 # Wait for Kubeapps Pods
-k8s_wait_for_pod_ready kubeapps app=kubeapps-ci
-k8s_wait_for_pod_ready kubeapps app=kubeapps-ci-internal-apprepository-controller
-k8s_wait_for_pod_ready kubeapps app=kubeapps-ci-internal-chartsvc
-k8s_wait_for_pod_ready kubeapps app=kubeapps-ci-internal-tiller-proxy
-k8s_wait_for_pod_ready kubeapps app=mongodb
-k8s_wait_for_pod_completed kubeapps apprepositories.kubeapps.com/repo-name=stable
+k8s_wait_for_deployment kubeapps kubeapps-ci
+echo "Deployment kubeapps-ci ready"
+k8s_wait_for_deployment kubeapps kubeapps-ci-internal-apprepository-controller
+echo "Deployment kubeapps-ci-internal-apprepository-controller ready"
+k8s_wait_for_deployment kubeapps kubeapps-ci-internal-chartsvc
+echo "Deployment kubeapps-ci-internal-chartsvc ready"
+k8s_wait_for_deployment kubeapps kubeapps-ci-internal-tiller-proxy
+echo "Deployment kubeapps-ci-internal-tiller-proxy ready"
```

Comment:
> echo "Deployment kubeapps-ci-internal-tiller-proxy ready"

Isn't this part of the output of kubectl rollout? I am wondering if this `echo` is redundant.

Reply:
yes, I added the log entry to ensure that there is no error between the `rollout` execution and the end of the function, just in case we introduce a typo while checking the exit code